controller/array_action/config.py

```diff
@@ -8,3 +8,6 @@
 
 # volume context
 CONTEXT_POOL = "pool"
+
+SRC_OBJECT_TYPE_VOLUME = 1
```

Comment:
I don't think the solution is just to remove it (it's not the first or last time this happens).

please add(/open a ticket to add) "allow-global-unused-variables=no" to our pylint configuration.

Reply:
when we find a gap in review, if we know of an easy way to prevent it from re-happening in the future, we include it as part of the review.
the usual way is adding a unit test to cover it, but this time it's even simpler!
and no part of the project is exclusive to a single contributor, on the contrary - we should all acquire experience in all parts, especially in a general linter.

Reply:
CSI-1660